_clients/data-prepper/data-prepper-reference.md

```diff
@@ -92,6 +92,42 @@ awsRegion | Conditionally | String | Represents the AWS Region to use ACM or S3.
 authentication | No | Object | An authentication configuration. By default, an unauthenticated server is created for the pipeline. This uses pluggable authentication for HTTPS. To use basic authentication, define the `http_basic` plugin with a `username` and `password`. To provide customer authentication, use or create a plugin that implements [GrpcAuthenticationProvider](https://github.com/opensearch-project/data-prepper/blob/main/data-prepper-plugins/armeria-common/src/main/java/com/amazon/dataprepper/armeria/authentication/GrpcAuthenticationProvider.java).
 
 
+### s3
+
+This is a source plugin that reads events from S3 objects.
+
+Option | Required | Type | Description
+:--- | :--- | :--- | :---
+notification_type | Yes | String | Must be `sqs`
+compression | No | String | The compression algorithm to apply; `none`, `gzip`, or `automatic`. Default is `none`.
```

Comment:
In the description, in order to match the structure of the other descriptions, should it be "The compression algorithm to apply. Can be..."? If not, the semicolon should just be a colon.

Reply:
Let's go with "The compression algorithm to apply. Can be 'none', 'gzip', or 'automatic'."